extensions/swift-package-index/src/search.tsx

```diff
@@ -101,7 +101,7 @@ async function performSearch(query: string, page: number, signal: AbortSignal):
   params.append("query", query);
   params.append("page", page.toString());
 
-  const response = await fetch("https://swiftpackageindex.com/api/search?" + params.toString(), {
+  const response = await fetch("https://spi-proxy-worker.dev-capturecontext-8f5.workers.dev/search?" + params.toString(), {
```

Comment:
Proxy is used to inject an SPI API key
proxy-worker source code is trivial and can be found here https://github.com/CaptureContext/spi-proxy-worker